_posts/zh/newsletters/2023-11-15-newsletter.md

```diff
@@ -19,7 +19,7 @@ lang: zh
 
 ## 田野调查：一段 Miniscript 旅程
 
-{% include articles/wizardsardine-miniscript.md extrah="#" %} {% assign timestamp="20:17" %}
```

Comment:
Looks right, can you please also remove throughout? Thank you @hulatown !

Reply:
Done